### PR TITLE
Test: Update all tests to use testing library

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Ellipsis/src/Ellipsis.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ export interface EllipsisProps extends React.ComponentPropsWithRef<'div'> {
   className?: string;
   /** Add a className for the ellipsis container div */
   containerClassName?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
```
Comment (franckgaudin) [inline packages/Ellipsis/src/Ellipsis.tsx:16]: 👍🏻

File: utils/bootstrap/helpers/template.mjs [hunk 2]
```diff
@@ -9,11 +9,13 @@ import './${name}.scss';
 
 export interface ${pascalCaseName}Props extends React.ComponentProps<'div'> {
   children: React.ReactNode;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
```
Comment (franckgaudin) [inline utils/bootstrap/helpers/template.mjs:13]: 👍🏻

**Comment by franckgaudin** [comment]
Good job, just missing the changeset (patch) to reference the changes in the changelog.

**Comment by vicky-comeau** [comment]
> Good job, just missing the changeset (patch) to reference the changes in the changelog.

Oops, thanks, I just added one :)

**Review by franckgaudin — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**